WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -265,6 +266,11 @@
     <hr>
     <%= f.input :remarks, disabled: is_actually_confirmed %>
 
+    <% if @competition.new_record? && @competition.cloned_from && !@competition.cloned_from.competition_tabs.empty? %>
```

Comment:
Would it be wrong to do `<% if @competition. clone_tabs %>` here?

You'd have to set it to false/nil appropriately.